compiler/ir/backend.py/src/org/jetbrains/kotlin/ir/backend/py/transformers/irToPy/IrDeclarationToPyTransformer.kt

```diff
@@ -34,9 +34,37 @@ class IrDeclarationToPyTransformer : BaseIrElementToPyNodeTransformer<List<stmt>
     }
 
     override fun visitField(declaration: IrField, context: PyGenerationContext): List<stmt> {
+        val fieldName = context.getNameForField(declaration).ident.toValidPythonSymbol()
+
+        if (
+            declaration.initializer != null &&
+            // todo: the following lines omit initializing of known stdlib fields that are unsupported yet. need to remove them eventually
+            declaration.symbol.owner.name.asString().toValidPythonSymbol() !in setOf(
+                "UNDEFINED_RESULT",
+                "output",
```

Comment:
@krzema12, jfyi, if you want to support any of those global properties in stdlib (including `output`), don't forget to remove them from here, otherwise they won't work